src/protocols/memory/ddr3.stanza

```diff
@@ -0,0 +1,573 @@
+doc: \<DOC>
+@brief Double Data Rate 3 Memory Standard (DDR3)
+
+DDR3 is a high speed memory protocol
+@see https://en.wikipedia.org/wiki/DDR3_SDRAM
+
+This functions and definitions in this file support defining DDR3
+connections between microprocessors and memories in a board design.
+
+This file contains curated values for skew and loss of DDR3 channel
+It specifies the bounds on the intra-pair
+skew timing and maximum loss as expected by this particular standard. The values
+returned are a toleranced value with upper/lower limits for the
+intra-pair skew and the maximum loss as a double representing dB. Some defaults in the
+table are derived from the references listed below.
+
+1. Skew match between CMD/ADDR/CTRL and CK.P is 0.0 +/- 25e-12 (25 ps)
+2. Skew match of CK is 0.0 +/- 1.0e-12 (1 ps)
+3. Skew match of DQS is 0.0 +/- 1.0e-12 (1 ps)
+4. Skew match between DQ and DQS.P within a byte lane is 0.0 +/- 3.5e-12 (3.5 ps)
+5. Skew match between CK.P and DQS.P is -75 ps to 900 ps (example values, needs verification)
+
+<DOC>
+
+#use-added-syntax(jitx)
+defpackage jsl/protocols/memory/ddr3:
+  import core
+  import jitx
+  import jitx/commands
+
+  import jsl/bundles
+  import jsl/errors
+  import jsl/ensure
+  import jsl/si
+  import jsl/pin-assignment
+
+doc: \<DOC>
+@brief DDR3 width enums
+This is a fixed list of possible lane widths for DDR3 memory channels
+@member DDR3-x8   8 bit width
+@member DDR3-x16 16 bit width
+<DOC>
+public pcb-enum jsl/protocols/memory/ddr3/DDR3-width:
+  DDR3-x8
+  DDR3-x16
+
+doc: \<DOC>
+@brief DDR3 Width enum conversion to integers
+This converts possible lane widths for ddr3 memory channels into the corresponding integer
+<DOC>
+public defn width-to-int (en:DDR3-width) -> Int:
+  switch(en) :
+    DDR3-x8  :  8
+    DDR3-x16 : 16
+
+doc: \<DOC>
+@brief DDR3 Width enum conversion to lane count
+This converts possible lane widths for ddr3 memory channels
+into the corresponding integer number of 8-bit lanes which
+determines the number of DQS pairs required.
+<DOC>
+public defn width-to-lane (en:DDR3-width) -> Int:
+  switch(en) :
+    DDR3-x8  : 1
+    DDR3-x16 : 2
+
+doc: \<DOC>
+@brief ddr3 topology
+This is a fixed list of possible topologies for ddr3 memories
+@member DDR3-FlyBy
+@member DDR3-ClamShell
```

Comment:
In the DDR4 code you call out this isn't supported. Do we support it in DDR3 ?

Reply:
Nope. Removing...